src/metabase/driver/firebolt.clj

```diff
@@ -133,12 +133,7 @@
 (defmethod sql.qp/date [:firebolt :year-of-era] [_ _ expr] (extract "year" expr))
 ; account for start of week setting in the :day-of-week implementation
 (defmethod sql.qp/date [:firebolt :day-of-week] [_ _ expr]
-  (let [offset (driver.common/start-of-week-offset :firebolt)]
-    (if (not= offset 0)
-      (extract "isodow" (sql.qp/add-interval-honeysql-form :firebolt expr (- offset 7) :day))
-      (extract "isodow" expr)
-      )
-    ))
+  (sql.qp/adjust-day-of-week :firebolt (extract "isodow" expr)))
```

Comment:
Do we need to do something [similar to Postgres](https://github.com/metabase/metabase/blob/4a1327283b2abebf1ee7eccdab638e448c178d5d/src/metabase/driver/postgres.clj#L403-L406) here? Is our day of week correct?

Reply:
That's why we use `isodow` here and not `dow`
From our docs
dow - Extract the day of the week as Sunday (0) to Saturday (6).
isodow - Extract the day of the week as Monday (1) to Sunday (7).
https://docs.firebolt.io/sql_reference/functions-reference/date-and-time/extract.html